skimage/filters/tests/test_gabor.py

```diff
@@ -77,6 +77,18 @@ def match_score(image, frequency):
     assert responses[1, 1] > responses[1, 0]
 
 
+def test_morlet_zero_sum():
```

Comment:
It is modelled after a test [just above](https://github.com/scikit-image/scikit-image/pull/2152/files#diff-3c8fefb50d9124a60ce4329a9cdc24f6R48) which creates double as many filters (not that this is the overhead) and seems to be fine time wise. The whole file seems to run through in around `.76s`